health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/editVillagePopulationPopUP.js

```
@@ -1,79 +1,88 @@
 
-  // State to manage confirmed population and target population
-  const [confirmedTotalPopulation, setConfirmedTotalPopulation] = useState("");
-  const [confirmedTargetPopulation, setConfirmedTargetPopulation] = useState("");
+  const [confirmedValues, setConfirmedValues] = useState({});
+  const [showComment, setShowComment] = useState(false);
+  const [errors, setErrors] = useState({});
   const [showToast, setShowToast] = useState(null);
+  const [updatedCensus, setUpdatedCensus] = useState(null); // Add updatedCensus as state
+
+  let workflowAction = "";
+  if (userRoles && userRoles.includes('POPULATION_DATA_APPROVER')) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)